src/playground/Widgets/ProgressSection/index.tsx

```diff
@@ -49,6 +51,7 @@ export function ProgressSection({
         </div>
       </div>
       <CategoryBar data={data} {...categoryBarProps} />
+      <p className="text-grey-50">{bottomComment}</p>
```

Comment:
I merged the color tokens a few hours ago (😬), so that color probably won't apply correctly. It would be great if you could merge main and check it out for yourself, but the token would be `text-f1-foreground-secondary`.

Reply:
oh ok, I noticed new colors when I did this, but that one I think didn't exist yet, I'll check it out now